webapp/app.yaml

```diff
@@ -11,15 +11,12 @@ builtins:
 
 handlers:
   # Couple of special-case dynamic components.
-- url: /components/wpt-env-flags.html
+- url: /components/wpt-env-flags.js
```

Comment:
We can simply remove this entry and fall back to the default.

Reply:
Not quite; it's a template-generated compontent (dynamic). Needs to be handled server-side.

Reply:
Oh sorry I didn't see the next entry (for `/components`). nvm then.